src/renderer/features/contacts/EditContactForm/ui/EditContactNavigation.tsx

```diff
@@ -13,5 +13,5 @@ export const EditContactNavigation = ({ contactId }: Props) => {
     navigate(createLink(Paths.EDIT_CONTACT, {}, { id: [contactId] }));
   };
 
-  return <IconButton size={16} name="edit" className="m-3" onClick={navigateToEdit} />;
+  return <ButtonIcon size="md" icon="edit" className="m-3" onClick={navigateToEdit} />;
```

Comment:
Shouldn't it be sm?

Reply:
It became a bigger one - `20px + background`
<img width="1047" alt="image" src="https://github.com/novasamatech/nova-spektr/assets/8149541/6cfa8f51-c610-4616-b964-47ed121ac3ae">